### DISCUSSION TITLE
Trying to send audio recorded by Expo to Azure Speech to Text API

### DISCUSSION THREAD

**Discussion by Rodrigo-N-Duarte** [expo/expo#26066] [Archived: Questions]
I'm trying to send an audio saved by expo-av to the Microsoft Azure speech to text api, in this function I receive the uri and search for the audio using it in base64, after that I transform it into binary with the buffer and then send it in the request . The answer is always the same, a 200 OK that everything went fine but the displayText, which is the audio recognition property, always comes as an empty string "". This method send my request:

![image](https://github.com/expo/expo/assets/103392533/f9fc6417-3a39-43ad-a6ae-f91f522c4b19)

I believe the problem is in the buffer sent, since expo does not have the functionality to read binary files, which requires conversion

**Comment by Alomgir27** [comment]
Hi @Rodrigo-N-Duarte , Is your problem solved?

**Reply by Rodrigo-N-Duarte** [reply]
I'd switched from Azure to OpenAI Speech to Text API, can't make work it with Microsoft API. But the question is still open for me.
